kfdef/kfctl_gcp_iap.yaml

```diff
@@ -342,14 +331,9 @@ spec:
       parameters:
       - name: namespace
         value: istio-system
-      - initRequired: true
-        name: ipName
+      - name: ipName
```

Comment:
Can we comment out ipname and add back the comment that says it will be auto-filled.

It should actually be a bug if we overwrite an explicitly defined value.

Reply:
done.